docs/REQUIREMENTS.md

```diff
@@ -1,119 +1,185 @@
 
-The items listed below are designed to create a Highly Available (HA) infrastructure when creating your cluster on bare-metal machines or in a vCenter/vSphere environment. All of these items are REQUIRED as listed.
+The items listed below are required for a Highly Available (HA) infrastructure when creating your Kubernetes cluster on physical machines, on generic Linux VMs, or in a VMware vCenter/vSphere environment. All of these items are REQUIRED as listed.
 
 Table of Contents
 
 - [Open Source Kubernetes Infrastructure Requirements for High Availability](#open-source-kubernetes-infrastructure-requirements-for-high-availability)
   - [Operating System](#operating-system)
   - [Machines](#machines)
-    - [VMware vSphere](#vmware-vsphere)
+    - [VMware vSphere or vCenter](#vmware-vsphere-or-vcenter)
       - [Resources](#resources)
       - [Machine Template Requirements](#machine-template-requirements)
-    - [Bare-Metal](#bare-metal)
+    - [Physical Machines or Linux VMs](#physical-machines-or-linux-vms)
   - [Network](#network)
     - [CIDR Block](#cidr-block)
     - [Static IP Addresses](#static-ip-addresses)
     - [Floating IP Addresses](#floating-ip-addresses)
+  - [Storage](#storage)
+    - [Configuration and Restrictions](#configuration-and-restrictions)
+    - [Bare metal and virtual machine storage example](#bare-metal-and-virtual-machine-storage-example)
+    - [Terraform storage example](#terraform-storage-example)
   - [Examples](#examples)
-    - [vCenter/vSphere Sample `tfvars` File](#vcentervsphere-sample-tfvars-file)
-    - [Bare Metal Sample `inventory` File](#bare-metal-sample-inventory-file)
+    - [vCenter/vSphere Sample tfvars File](#vcentervsphere-sample-tfvars-file)
+    - [Physical Machine or VM Sample Inventory File](#physical-machine-or-vm-sample-inventory-file)
   - [Deployment](#deployment)
   - [Third-Party Tools](#third-party-tools)
 
 ## Operating System
 
-An Ubuntu Linux operating system is required for the tasks associated with standing up infrastructure using the tools in this repository.
+An Ubuntu Linux operating system is required for the machine that uses the tools in this repository to perform the tasks associated with standing up infrastructure for a SAS Viya deployment.
 
-| OS | Description |
+| Operating System | Description |
 | --- | --- |
-| [Ubuntu 20.04 LTS](https://releases.ubuntu.com/20.04/) | You must have a user account and password with privileges that enable unprompted `sudo`. You must also have a shared "private/public" SSH key pair to use with each system. These are required for the Ansible tools that are described below. |
+| [Ubuntu 22.04 LTS](https://releases.ubuntu.com/22.04/) | You must have a user account and password with privileges that enable unprompted `sudo`. You must also have a shared "private/public" SSH key pair to use with each system. These are required for the Ansible tools that are described below. |
```

Comment:
The Readme content lists Ubuntu 20.04 LTS and 22.04 LTS as being supported but this file says 22.04 LTS is required. Should 20.04 LTS be listed here also?